recipe/meta.yaml

```diff
@@ -62,55 +61,29 @@ requirements:
     - sed       # [unix]
   host:
     - msinttypes  # [win and vc<14]
-    - bzip2
-    - libcurl
-    - zlib
-    - hdf4
-    - hdf5
-    - jpeg
-    - libzip
+    - bzip2 1.0.8
+    - libcurl 7.88.1
+    - zlib {{ zlib }}
+    - hdf4 4.2.13
+    - hdf5 1.12.1
+    - jpeg 9e
+    - libzip 1.8.0
+    - openssl {{ openssl }}  # [not win]
```

Comment:
Maybe to me this is still unclear, but why is the pinning inherited from `cbc.yaml` in the host section if this has to be built agains `openssl 3.*`?

Reply:
it's using an aggregate branch called openssl3_builds (see the abs.yaml). In this branch, the aggregate cbc.yaml has the openssl 3 pinning

Reply:
gotcha. So I guess it's ok to keep this with the knowledge that aggregate will soon be update, right?

Reply:
Well it's kind of a question of the tradeoffs/approach that @JeanChristopheMorinPerso has chosen. Also related to the discussion [here](https://github.com/AnacondaRecipes/cyrus-sasl-feedstock/pull/5)

Reply:
Yeah. Technically it's a really bad practice, but unfortunately it's a practice that the distro team has been doing for a while. We've decided to keep doing it for now and discuss it later. There is already some talks in progress regarding how we pin packages, for example to get rid of implicit dependencies, etc but it'll take while I think before we can reach a consensus.

Reply:
As far as i understand, Jean-Christophe isn't going to merge to master until the aggregate is updated

Reply:
@JeanChristopheMorinPerso just spoke to Rafael, he said that you are doing a mutual staging branch thing between you and the ICU rebuild, then when the packages are built out you'll merge the branches containing the updated aggregate pinnings for openssl and icu, then merge the staging branches? So then, the aggregate cbc.yaml pinning will be updated before the updated feedstocks are merged.

I'm also wondering why we wouldn't just merge the updated aggregate/cbc.yaml first, then rebuild and merge the packages one by one?

Reply:
>he said that you are doing a mutual staging branch thing between you and the ICU rebuild

Umm, I suggested that we could collaborate on Qt to avoid having to rebuild Qt multiple times (since Qt is an heavy and painful package).

@ryanskeith can add more details, but we wanted to be extra careful. We didn't want to force OpenSSL 3 on everyone too early. We are planning on merging the aggregate branch in master on Monday.

Reply:
that makes sense intuitively, I'm just wondering if it really makes a difference, since everything that depends on openssl is being rebuilt anyway. And then if someone updates one of those packages before you can get to it, they've done the work for you?